package.json

```diff
@@ -33,26 +33,28 @@
     "bignumber.js": "^8.1.1",
     "bootstrap": "^4.3.1",
     "core-js": "^2.5.4",
+    "crypto": "^1.0.1",
     "execa": "^2.0.4",
     "font-awesome": "^4.7.0",
     "lodash": "^4.17.15",
-    "ng2-completer": "^2.0.8",
+    "ng": "0.0.0",
```

Comment:
What is this `ng` used for?

Reply:
i'll remove passphrase, enckey,
"ng" is mistake,  i'll restore it to "ng2-completer"